ci/forge/Dockerfile.manylinux

```diff
@@ -0,0 +1,53 @@
+# syntax=docker/dockerfile:1.3-labs
+
+FROM quay.io/pypa/manylinux2014_x86_64:2022-12-20-b4884d9
+
+ENV BUILD_JAR=1
+ENV RAY_INSTALL_JAVA=1
+
+RUN <<EOF
+
+set -exuo pipefail
+
+if [[ ! -e /usr/bin/nproc ]]; then
```

Comment:
the content of this is exactly the same as ci/forge/build-manylinux-forge.sh. We'll reuse the script once we convert this into wanda.

Reply:
I guess you can do it now. :)

Reply:
let's do it!